ngraph/core/include/ngraph/node.hpp

```diff
@@ -197,6 +201,11 @@ namespace ngraph
         /// \returns true if successful
         virtual bool evaluate(const HostTensorVector& output_values,
                               const HostTensorVector& input_values) const;
+        /// \brief Evaluates the op on input_values putting results in output_values
```

Comment:
I think comment should contain description what is EvaluateContext and when user should use this function

Reply:
added description for Evaluate argument. I suppose a detailed description should be done in the documentation.

Reply:
Yes, and it is also. I will update docs

Reply:
done